<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Amp_4_4_Mixer</class>
 <widget class="QDialog" name="Amp_4_4_Mixer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>215</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>215</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="AOut4_NIn4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut1_NIn4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>A-Out1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut1_AIn3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>A-Out2</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut2_NIn3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N-In2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut3_AIn2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut4_AIn3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut2_AIn2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut1_AIn2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut2_AIn4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut4_AIn2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut3_NIn2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut1_AIn1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut2_NIn4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut1_AIn4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut2_AIn1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut4_NIn2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut3_NIn4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut4_NIn3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>130</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>A-Out3</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut3_AIn4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut1_NIn1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut1_NIn3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>A-Out4</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut4_NIn1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut3_NIn3">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut2_NIn2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A-In1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut4_AIn4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N-In3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A-In4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut3_AIn3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut2_NIn1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut1_NIn2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A-In3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N-In1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut4_AIn1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>A-In2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>N-In4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut3_NIn1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut2_AIn3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AOut3_AIn1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>35</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Close">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
